src/app/rpmostree-dbus-helpers.c

```diff
@@ -29,6 +29,26 @@
 #include <signal.h>
 #include <systemd/sd-login.h>
 
+static gboolean opt_txn_quiet;
+
+static const GOptionEntry txn_common_option_entries[] = {
+  { "quiet", 'q', 0, G_OPTION_ARG_NONE, &opt_txn_quiet, "Quiet output; Primarily, do not emit progress information", NULL },
+  { NULL }
+};
+
+GOptionGroup *
+rpmostree_txn_common_options (void)
+{
+  static GOptionGroup *g;
```

Comment:
Hmm, do we really need to make this a static? We'll only be at most running this path once during an invocation, right?